crates/rattler_installs_packages/src/resolve.rs

```diff
@@ -167,10 +169,9 @@ impl<'db, 'i> PypiDependencyProvider<'db, 'i> {
         }
 
         // Filter only artifacts we can work with
-        artifacts.retain(|a| a.is::<Wheel>());
         if artifacts.is_empty() {
             // If there are no wheel artifacts, we're just gonna skip it
-            return Err("there are no wheels available");
+            return Err("there are no artifacts available");
```

Comment:
Maybe we should talk about `packages` instead? Not sure the average person understands what we mean with an artifact.

Reply:
Sure!